core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -192,64 +194,77 @@ private void maybeDeleteActiveConsumers(final String groupId,
+                    .get();
             } else {
                 throw new IllegalStateException("Consumer group '" + groupId + "' is still active "
-                        + "and has following members: " + members + ". "
-                        + "Make sure to stop all running application instances before running the reset tool."
-                        + " You can use option '--force' to remove active members from the group.");
+                    + "and has following members: " + members + ". "
+                    + "Make sure to stop all running application instances before running the reset tool."
+                    + " You can use option '--force' to remove active members from the group.");
             }
         }
     }
 
     private void parseArguments(final String[] args) {
         final OptionParser optionParser = new OptionParser(false);
-        applicationIdOption = optionParser.accepts("application-id", "The Kafka Streams application ID (application.id).")
-            .withRequiredArg()
-            .ofType(String.class)
-            .describedAs("id")
-            .required();
-        bootstrapServerOption = optionParser.accepts("bootstrap-servers", "Comma-separated list of broker urls with format: HOST1:PORT1,HOST2:PORT2")
+        applicationIdOption =
+            optionParser.accepts("application-id", "The Kafka Streams application ID (application.id).")
+                .withRequiredArg()
+                .ofType(String.class)
+                .describedAs("id")
+                .required();
+        bootstrapServerOption = optionParser
+            .accepts("bootstrap-servers", "Comma-separated list of broker urls with format: HOST1:PORT1,HOST2:PORT2")
             .withRequiredArg()
             .ofType(String.class)
             .defaultsTo("localhost:9092")
             .describedAs("urls");
-        inputTopicsOption = optionParser.accepts("input-topics", "Comma-separated list of user input topics. For these topics, the tool will reset the offset to the earliest available offset.")
+        inputTopicsOption = optionParser.accepts("input-topics",
+            "Comma-separated list of user input topics. For these topics, the tool will reset the offset to the earliest available offset.")
             .withRequiredArg()
             .ofType(String.class)
             .withValuesSeparatedBy(',')
             .describedAs("list");
-        intermediateTopicsOption = optionParser.accepts("intermediate-topics", "Comma-separated list of intermediate user topics (topics that are input and output topics, e.g., used in the deprecated through() method). For these topics, the tool will skip to the end.")
+        intermediateTopicsOption = optionParser.accepts("intermediate-topics",
+            "Comma-separated list of intermediate user topics (topics that are input and output topics, e.g., used in the deprecated through() method). For these topics, the tool will skip to the end.")
```

Comment:
We (Streams) usually break the arguments of a function call as follows:
```suggestion
        intermediateTopicsOption = optionParser.accepts(
            "intermediate-topics",
            "Comma-separated list of intermediate user topics (topics that are input and output topics, e.g., used in the deprecated through() method). For these topics, the tool will skip to the end."
        )
```

or 

```suggestion
        intermediateTopicsOption = optionParser.accepts(
            "intermediate-topics",
            "Comma-separated list of intermediate user topics (topics that are input and output topics, e.g., used in the deprecated through() method). For these topics, the tool will skip to the end.")
```
IMO both are better readable than having arguments filling up the line and then break. If all arguments fit in one line I would not break them. So my preferred formatting would be either all arguments fit in one line or each argument on its own line.